cms/static/cms/js/require-config.js

```diff
@@ -69,6 +69,7 @@
             "modernizr": "edx-pattern-library/js/modernizr-custom",
             "afontgarde": "edx-pattern-library/js/afontgarde",
             "edxicons": "edx-pattern-library/js/edx-icons",
+            "paging-collection": "edx-ui-toolkit/js/pagination/paging-collection",
```

Comment:
I'd prefer that folks import this with the full path, rather than defining a synonym for it. We have a lot of UI Toolkit files (and will be adding a lot more) so I don't want them to have to be managed individually.